doc/source/whatsnew/v2.3.0.rst

```diff
@@ -143,7 +143,7 @@ Period
 
 Plotting
 ^^^^^^^^
--
+- Bug in :meth:`DataFrame.plot.bar` with ``stacked=True`` where labels on stacked bars with zero-height segments were incorrectly positioned at the base instead of the label position of the previous segment (:issue:`59429`)
```

Comment:
This note should be move to v3.0.0.rst.

Reply:
2.3.0 is a "special" release in that we are not doing it from the main branch. We are only backporting certain PRs to the 2.3.0 branch.